src/nvim/option.c

```diff
@@ -218,20 +218,22 @@ typedef struct vimoption {
 #define P_RALL          0x6000U  /* redraw all windows */
 #define P_RCLR          0x7000U  /* clear and redraw all */
 
-#define P_COMMA         0x8000U  /* comma separated list */
-#define P_NODUP         0x10000U /* don't allow duplicate strings */
-#define P_FLAGLIST      0x20000U /* list of single-char flags */
-
-#define P_SECURE        0x40000U /* cannot change in modeline or secure mode */
-#define P_GETTEXT       0x80000U /* expand default value with _() */
-#define P_NOGLOB       0x100000U /* do not use local value for global vimrc */
-#define P_NFNAME       0x200000U /* only normal file name chars allowed */
-#define P_INSECURE     0x400000U /* option was set from a modeline */
-#define P_PRI_MKRC     0x800000U /* priority for :mkvimrc (setting option has
-                                    side effects) */
-#define P_NO_ML       0x1000000U /* not allowed in modeline */
-#define P_CURSWANT    0x2000000U /* update curswant required; not needed when
-                                  * there is a redraw flag */
+#define P_COMMA         0x8000U    ///< comma separated list
```

Comment:
By e1f782187449870b04301fc68d5f7e38d47a1faf, `flags` was changed `long_u` with `uint32_t` and `L` with `U`. Besides `n1124` reads that the `integer-suffix` of `U` (or `u`) can be represented `unsigned int`, `unsigned long int`, `unsigned long long int` for Hexadecimal Constant. So I think this is acceptable.